modules/psap-special-resource-operator-using-crd.adoc

```diff
@@ -0,0 +1,408 @@
+kind: SpecialResourceModule
+metadata:
+    name: acm-simple-kmod
+spec:
+    namespace: acm-simple-kmod
+    chart:
+        name: acm-simple-kmod
+        version: 0.0.1
+        repository:
+            name: acm-simple-kmod
+            url: cm://acm-simple-kmod/acm-simple-kmod-chart
+    set:
+        kind: Values
+        apiVersion: sro.openshift.io/v1beta1
+        buildArgs:
+            - name: "KMODVER"
+              value: "SRO"
+        registry: <your_registry>  <1>
+        git:
+            ref: master
+            uri: https://github.com/openshift-psap/kvc-simple-kmod.git
+    watch:
+            - path: "$.metadata.labels.openshiftVersion"
+              apiVersion: cluster.open-cluster-management.io/v1
+              kind: ManagedCluster
+              name: spoke1
+----
+<1> Specify the URL for a registry that you have configured.
+
+. Create the special resource module by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f charts/examples/acm-simple-kmod.yaml
+----
+
+.Verification
+
+. Check the status of the build pods by running the following command:
++
+[source,terminal]
+----
+$ KUBECONFIG=~/hub/auth/kubeconfig oc get pod -n acm-simple-kmod
+----
++
+.Example Output
+[source,terminal]
+----
+NAME                                                   READY   STATUS      RESTARTS   AGE
+acm-simple-kmod-4-18-0-305-34-2-el8-4-x86-64-1-build   0/1     Completed   0          42m
+----
+
+. Check that the policies have been created by running the following command:
++
+[source,terminal]
+----
+$ KUBECONFIG=~/hub/auth/kubeconfig oc get placementrules,placementbindings,policies -n acm-simple-kmod
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                                      AGE   REPLICAS
+placementrule.apps.open-cluster-management.io/acm-simple-kmod-placement   40m
+
+NAME                                                                         AGE
+placementbinding.policy.open-cluster-management.io/acm-simple-kmod-binding   40m
+
+NAME                                                                 REMEDIATION ACTION   COMPLIANCE STATE   AGE
+policy.policy.open-cluster-management.io/policy-acm-simple-kmod-ds   enforce              Compliant          40m
+----
+
+. Check that the resources have been reconciled by running the following command:
++
+[source,terminal]
+----
+$ KUBECONFIG=~/hub/auth/kubeconfig oc get specialresourcemodule acm-simple-kmod -o json | jq -r '.status'
+----
++
+.Example output
+
+[source,terminal]
+----
+{
+  "versions": {
+    "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6a3330ef5a178435721ff4efdde762261a9c55212e9b4534385e04037693fbe4": {
+      "complete": true
+    }
+  }
+}
+----
+
+. Check that the resources are running in the spoke by running the following command:
++
+[source,terminal]
+----
+$ KUBECONFIG=~/spoke1/kubeconfig oc get ds,pod -n acm-simple-kmod
+----
++
+Example output
+
+[source,terminal]
```

Comment:
@slovern Do you intend for this example output to be on its own level, or do you want it attached to the final verification step?

Reply:
Nope, it should be attached to the verification. I've corrected